modules/apps/blogs/blogs-test/src/testIntegration/java/com/liferay/blogs/service/test/BlogsEntryLocalServiceTest.java

```diff
@@ -648,13 +660,20 @@ public void testDeleteDiscussion() throws Exception {
 				BlogsEntry.class.getName(), entry.getEntryId()));
 	}
 
-	@Test(expected = NoSuchEntryException.class)
+	@Test
 	public void testDeleteEntry() throws Exception {
 		BlogsEntry entry = addEntry(false);
 
 		_blogsEntryLocalService.deleteEntry(entry);
 
-		_blogsEntryLocalService.getEntry(entry.getEntryId());
+		try {
+			_blogsEntryLocalService.getEntry(entry.getEntryId());
+
+			Assert.fail();
+		}
+		catch (Exception exception) {
+			Assert.assertTrue(exception instanceof NoSuchEntryException);
+		}
```

Comment:
Q: Why this instead of the `expected` attribute in the annotation?

Reply:
How do we know that the exception doesn't come from the previous line
```java
_blogsEntryLocalService.deleteEntry(entry);
```